### ISSUE TITLE
Error with Python 3.9

### ISSUE THREAD

**Issue by vsergeyev** [zapatacomputing/qe-qiskit#11]
It installs well, and  when trying to use `from qeqiskit.simulator import QiskitSimulator` got traceback:

```
Python 3.9.0 (default, Oct 27 2020, 14:13:35) 
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from qeqiskit.simulator import QiskitSimulator
/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/__init__.py:62: RuntimeWarning: Could not import the Aer provider from the qiskit-aer package. Install qiskit-aer or check your installation.
  warnings.warn('Could not import the Aer provider from the qiskit-aer '
/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/__init__.py:72: RuntimeWarning: Could not import the IBMQ provider from the qiskit-ibmq-provider package. Install qiskit-ibmq-provider or check your installation.
  warnings.warn('Could not import the IBMQ provider from the '
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/vserh/Documents/Temp/quantum/q/qe-qiskit/src/python/qeqiskit/simulator/__init__.py", line 1, in <module>
    from .simulator import QiskitSimulator
  File "/Users/vserh/Documents/Temp/quantum/q/qe-qiskit/src/python/qeqiskit/simulator/simulator.py", line 2, in <module>
    from qiskit import Aer, IBMQ, execute
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/__init__.py", line 79, in <module>
    from qiskit.execute import execute  # noqa
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/execute.py", line 26, in <module>
    from qiskit.compiler import transpile, assemble, schedule
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/compiler/__init__.py", line 35, in <module>
    from .transpile import transpile
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/compiler/transpile.py", line 23, in <module>
    from qiskit.transpiler import Layout, CouplingMap, PropertySet, PassManager
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/transpiler/__init__.py", line 61, in <module>
    from .runningpassmanager import FlowController
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/transpiler/runningpassmanager.py", line 23, in <module>
    from qiskit.dagcircuit import DAGCircuit
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/dagcircuit/__init__.py", line 39, in <module>
    from .dagcircuit import DAGCircuit
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/qiskit/dagcircuit/dagcircuit.py", line 30, in <module>
    import networkx as nx
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/networkx/__init__.py", line 114, in <module>
    import networkx.generators
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/networkx/generators/__init__.py", line 14, in <module>
    from networkx.generators.intersection import *
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/networkx/generators/intersection.py", line 13, in <module>
    from networkx.algorithms import bipartite
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/networkx/algorithms/__init__.py", line 16, in <module>
    from networkx.algorithms.dag import *
  File "/Users/vserh/Documents/Temp/quantum/q/venv/lib/python3.9/site-packages/networkx/algorithms/dag.py", line 23, in <module>
    from fractions import gcd
ImportError: cannot import name 'gcd' from 'fractions' (/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/fractions.py)
>>> 

```

**Comment by tirkarthi** [comment]
This seems to have been fixed in networkx : https://github.com/networkx/networkx/pull/2861

**Comment by max-radin** [comment]
Thanks @tirkarthi. It seems that the fix was introduced in networkx 2.4. Marking this issue as closed since `qe-qiskit` now has this version as dependency (via `z-quantum-core`).
